script.js

```diff
@@ -162,6 +211,7 @@ form.addEventListener("submit", function (event) {
 
   let errorMessage = document.getElementById("error-message");
   let emailInput = document.getElementById("email-input");
+  saveData();
```

Comment:
Kindly change this part so that saveData() will be triggered each time an input field changes value as the idea is to save values even when the form wasn't submitted 

**project requirements:**
You should implement the following interactions:
When the user changes the content of any input field, the data is saved to the local storage.